db/migrate/20220325131956_create_cars.rb

```diff
@@ -0,0 +1,17 @@
+class CreateCars < ActiveRecord::Migration[7.0]
+  def change
+    create_table :cars do |t|
+      t.string :brand 
+      t.string :model
+      t.integer :model_year
+      t.string :photo_url
+      t.text :description
+      t.boolean :reserved ,default: false
+      t.decimal :rent_fee
+      t.integer :likes_counter
+      t.integer :reservation_counter
```

Comment:
It will be better if the like_counter and reservation_counter are set to default 0, so it would not give a null value when there is no likes or reservation which might become a bug later.

Reply:
Thank you , I Fixed this issue 👍